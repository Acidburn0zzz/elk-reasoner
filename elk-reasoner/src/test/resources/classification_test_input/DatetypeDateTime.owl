Prefix(xsd:=<http://www.w3.org/2001/XMLSchema#>)
Prefix(owl:=<http://www.w3.org/2002/07/owl#>)
Prefix(:=<http://www.semanticweb.org/TestELK.owl#>)
Prefix(xml:=<http://www.w3.org/XML/1998/namespace>)
Prefix(rdf:=<http://www.w3.org/1999/02/22-rdf-syntax-ns#>)
Prefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)


Ontology(<http://www.semanticweb.org/TestELK.owl>

Declaration(Class(:A))
Declaration(Class(:B))
Declaration(Class(:C))
Declaration(Class(:D))
Declaration(Class(:E))

EquivalentClasses(:C DataSomeValuesFrom(:prop xsd:dateTime))
EquivalentClasses(:D DataSomeValuesFrom(:prop xsd:dateTimeStamp))
EquivalentClasses(:A DataHasValue(:prop "1956-06-25T04:00:00-05:00"^^xsd:dateTime))
EquivalentClasses(:B DataHasValue(:prop "1956-06-25T10:00:00+01:00"^^xsd:dateTime))
EquivalentClasses(:E DataHasValue(:prop "1956-06-25T10:00:00"^^xsd:dateTime))

Declaration(DataProperty(:prop))
)