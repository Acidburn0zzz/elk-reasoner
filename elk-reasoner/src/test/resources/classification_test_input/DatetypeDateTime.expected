Prefix(xsd:=<http://www.w3.org/2001/XMLSchema#>)
Prefix(owl:=<http://www.w3.org/2002/07/owl#>)
Prefix(:=<http://www.semanticweb.org/TestELK.owl#>)
Prefix(xml:=<http://www.w3.org/XML/1998/namespace>)
Prefix(rdf:=<http://www.w3.org/1999/02/22-rdf-syntax-ns#>)
Prefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)


Ontology(<http://www.semanticweb.org/TestELK.owl>

Declaration(Class(:A))
Declaration(Class(:B))
Declaration(Class(:C))
Declaration(Class(:D))
Declaration(Class(:E))

SubClassOf(<http://www.semanticweb.org/TestELK.owl#C> owl:Thing)
SubClassOf(<http://www.semanticweb.org/TestELK.owl#A> <http://www.semanticweb.org/TestELK.owl#D>)
SubClassOf(<http://www.semanticweb.org/TestELK.owl#B> <http://www.semanticweb.org/TestELK.owl#D>)
SubClassOf(<http://www.semanticweb.org/TestELK.owl#D> <http://www.semanticweb.org/TestELK.owl#C>)
SubClassOf(<http://www.semanticweb.org/TestELK.owl#E> <http://www.semanticweb.org/TestELK.owl#C>)

Declaration(DataProperty(:prop))
)